.gitignore

```diff
@@ -2,4 +2,6 @@
 .idea
 .project
 .settings/
+.theia/
```

Comment:
These changes shouldn't be in this PR right? They are in the previous PR. You should rebase.